### DISCUSSION TITLE
Possibly integrate mpd (music player daemon) with current hyprdots?

### DISCUSSION THREAD

**Discussion by ghost** [prasanthrangan/hyprdots#645] [Ideas]
Been working for the past 4 hours on getting mpd with ncmpcpp and waybar working, and it works well! I have no idea if prasanthrangan would want this merged in some way in this repo, but @prasanthrangan if you do, i'll submit a pull request.

![image](https://github.com/prasanthrangan/hyprdots/assets/139541046/ce83ba10-ce08-4696-93e3-03ad640371d1)

This is still WIP, i might even try to get spotify integration, but it's at least got to a point where i'm happy.

**Comment by Ramdonx** [comment]
Hello, can you pls share this config?
